src/main/java/com/rtb/projectmanagementtool/auth/AuthOps.java

```diff
@@ -27,18 +27,25 @@ public AuthOps(DatastoreService ds) {
     this.controller = new UserController(ds);
   }
 
-  public String getAuthID() {
-    // call auth service to generate AuthID
-    UserService userService = UserServiceFactory.getUserService();
-    String AuthID = userService.getCurrentUser().getUserId();
-    return AuthID;
+  public String getAuthID(HttpServletRequest request, HttpServletResponse response) {
+    if (whichUserIsLoggedIn(request, response) != -1) {
+      // call auth service to generate AuthID
+      UserService userService = UserServiceFactory.getUserService();
+      String AuthID = userService.getCurrentUser().getUserId();
+      return AuthID;
+    }
+    // if no one logged in, return empty string
+    return "";
   }
 
   // get/create cookie and set value to userID
-  public void createAndSetCookieNewUser(HttpServletRequest request, long userID) {
```

Comment:
nit:  this method would be useful for other use cases as well, and not just for the new user use case.  It would be ok to rename this to something more generic like "setLoggedInCookie"